packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -77,6 +77,10 @@ const secondaryButtonStyles = ({
   ${theme.mq.untilKilo} {
     margin-right: 0;
     margin-top: ${theme.spacings.mega};
+    &:hover {
+      background-color: transparent;
+      border-color: transparent;
+    }
```

Comment:
The hover state looks correct now, however, the active state still has the wrong styles.

<img width="1292" alt="Screenshot 2023-02-06 at 15 10 55" src="https://user-images.githubusercontent.com/11017722/216995920-82bd337a-58f9-4411-8b6f-b76baa5d2ccb.png">

I wonder if it would be better to add these overrides in the [`Button` component](https://github.com/sumup-oss/circuit-ui/blob/main/packages/circuit-ui/components/Button/Button.tsx#L297-L306) to ensure that they stay in sync with any other changes that are made there. What do you think?